src/librustc/ty/mod.rs

```diff
@@ -1170,7 +1170,8 @@ pub struct ParameterEnvironment<'tcx> {
     /// Obligations that the caller must satisfy. This is basically
     /// the set of bounds on the in-scope type parameters, translated
     /// into Obligations, and elaborated and normalized.
-    pub caller_bounds: Vec<ty::Predicate<'tcx>>,
+    /// This slice is interned.
```

Comment:
I don't mind it, but I think that `&'tcx [...]` *implies* interning, so you don't really need to say it explicitly. (That is, you can't really get a reference with that lifetime unless it is interned, or at least allocated in a tcx arena.)